docker/include/bin/start_faux

```diff
@@ -141,6 +141,26 @@ if [ -n "${options[ntp_client]}" ]; then
     java -jar NTPClient/build/libs/NTPClient-1.0-SNAPSHOT.jar "time.google.com" "123" "3" &
 fi
 
+# ntp_pass queries the NTP server learnt from DHCP. ntp_fail sends to time.google.com
+if [ -n "${options[ntp_pass]}" ] || [ -n "${options[ntp_fail]}" ]; then
```

Comment:
NB, can be written: 

[ -n "${options[ntp_pass]}" -o -n "${options[ntp_fail]}" ];

The current thing you have is ok, but is mixing boolean logic layers, essentially extra redundant parens.

Reply:
updated, thanks.